Automattic/jetpack
# Multisite: remove commented out code, fix linting warnings & errors

#### Changes proposed in this Pull Request:

* This does not change any feature. We're just getting rid of commented out code that's been there for many versions, and fixing all PHPCS linting errors and warnings in the process so we can remove that file from our PHPCS exclusion list.

#### Jetpack product discussion

* Internal reference: p3btAN-1xB-p2

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Start from a multisite network when Jetpack is network-activated.
* Go to the primary site and connect it to WordPress.com. Enable Protect.
* Go to Network Admin > Jetpack > Settings, and check that everything is displayed properly.
